### ISSUE TITLE
Search Result IE -> results at the bottom

### ISSUE THREAD

**Issue by critzinger** [owncloud/core#21898]
Hello everybody,

my English is not perfect.

I have two ownCloud installations with the same problem in displaying the search results in IE.

Technical data of the installations:

Web: Apache2.4
PHP: 5.6
DB: MySQL
OC Version: 8.2.2
IE version: 11

In both versions, I have the problem, that the search results are displayed in Internet Explorer all the way down (partially outside the viewing area).
I've attached two screenshots with the identical search in IE and Firefox.

Does anyone know a way to configure the search so that the results are also displayed in IE directly below?

![owncloudsuchefirefox](https://cloud.githubusercontent.com/assets/16885644/12561483/cd89d154-c3a0-11e5-9896-ad924a88a13f.JPG)
![owncloudsucheie](https://cloud.githubusercontent.com/assets/16885644/12561484/cd8aaf02-c3a0-11e5-8f36-bd00604753dd.JPG)

**Comment by LukasReschke** [comment]
@owncloud/designers

**Comment by MorrisJobke** [comment]
Indeed - broken, but only in IE9, IE10 and IE11 - edge works fine.

**Comment by Pookay** [comment]
Hi,
on 8.2.2
modifying owncloud/apps/css/files.css line 60

```
/* fit app list view heights */
.app-files #app-content>.viewcontainer {
        min-height: 100%;
}

```
to `min-height: 0%` seems to resolve the problem. But it may cause another one, I guess.

**Comment by MorrisJobke** [comment]
@Pookay Could I ask you to send this in as a PR? There is also a web based way to send in these patches: https://github.com/blog/1945-quick-pull-requests

**Comment by MorrisJobke** [comment]
Fixed with #22004 in 9.0

**Comment by MorrisJobke** [comment]
Thanks @Pookay for your contribution - you are very welcome in our #owncloud-dev channel on free node and on the devel mailing list https://mailman.owncloud.org/mailman/listinfo/devel
